resources/js/components/Kiosk.vue

```diff
@@ -440,13 +440,23 @@ export default {
                     this.hasError = false;
                     let name = (response.data.name ? response.data.name : "Unknown User");
                     let direction = response.data.punch;
-                    let swalText = (direction === 'in') ? 'Nice to see you, ' + name + '.' : 'Have a great day, ' + name + '!';
+                    let swalText, swalIcon;
+
+                    if (response.data.message) {
+                        swalText = (direction === 'in') ? `Nice to see you, ${name}.` : `Have a great day, ${name}!`;
```

Comment:
move the definition of `swalText` outside the if statement since it's the same in the `if` and `else` blocks

Reply:
Meant to clean that up - was fiddling with what text would go where and forgot to put that back...will fix.